src/components/Router/StepsRouter.tsx

```diff
@@ -65,6 +66,9 @@ class StepsRouter extends Component<StepsRouterProps> {
       ? hideOnfidoLogo
       : globalUserOptions.enterpriseFeatures?.hideOnfidoLogo && hideOnfidoLogo
 
+    // Motion Web SDK already includes a footer
+    const hideFooterLogic = step.type === 'motion'
```

Comment:
I feel that there are too many `step.type === 'motion'`. We could leverage `PrivateStepConfig` and having a more generic way to handle the specific display (transparent + edgeToEdge + hideFooter). WDYT?

Reply:
I was not aware of this interface. Yeah, I think that'd work!